app/models/channel.rb

```diff
@@ -1,4 +1,8 @@
 class Channel < ActiveRecord::Base
   has_and_belongs_to_many :text_components
   belongs_to :report
+
+  def self.default(report)
+    find_by(name: "sensorstory", report: report)
```

Comment:
Although the constant will not work. The constant will be initialized when the application starts. If for any reason the database changes, this changes will go unnoticed (hello breaking tests).

I will introduce a scope:

```
  scope :chatbot,     -> { find_by(report: Report.current, name: "chatbot") }
  scope :sensorstory, -> { find_by(report: Report.current, name: "sensorstory") }
```

which can then be used by:

```
Channel.chatbot
```